### PR TITLE
Context and scope

### PR DESCRIPTION
**Pull request by UO276188**
Updated part 3.2 of the documentation

### CODE REVIEW

File: docs/03_system_scope_and_context.adoc [hunk 1]
```diff
@@ -58,8 +67,13 @@ E.g. UML deployment diagram describing channels to neighboring systems,
 together with a mapping table showing the relationships between channels and input/output.
 
 ****
+The system will follow the SOLID principles by which the user's privacy will be preserved.
+This means non storing personal information and using "pods" (Personal Online Data Store), where the customer's personal dara is stored.
```
Comment (CarolinaUniovi) [inline docs/03_system_scope_and_context.adoc:71]: "personal dara" instead of "personal data"

File: docs/03_system_scope_and_context.adoc [hunk 2]
```diff
@@ -58,8 +67,13 @@ E.g. UML deployment diagram describing channels to neighboring systems,
 together with a mapping table showing the relationships between channels and input/output.
 
 ****
+The system will follow the SOLID principles by which the user's privacy will be preserved.
+This means non storing personal information and using "pods" (Personal Online Data Store), where the customer's personal dara is stored. 
 
+All the application will be developed by using Typescript and the React framewrok,
```
Comment (CarolinaUniovi) [inline docs/03_system_scope_and_context.adoc:73]: "framewrok" instead of "framework"

**Review by CarolinaUniovi — COMMENTED** [review]
Pointing two typo errors.

**Review by UO277653 — APPROVED** [review]
After taking a look at the documentation and the diagrams I think that it is OK!

**Pull request merged**